Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -256,3 +257,50 @@ export function updateState(widgetState) {
     yRotationLength
   );
 }
+
+export function correctPlaneViewUp(
+  initialPlaneSource,
+  computedPlane,
+  viewType
+) {
```

Comment:
it does not only correct view up, it also rotate the view plane.
Suggestion: 
```
/**
 * First rotate planeToTransform to match targetPlane normal.
 * Then rotate around targetPlane normal to preserve world plane "up" vector (i.e. Origin->p2 ).
 * There is an infinite number of options to rotate a plane normal to another. Here we attempt to
 * do it by preserving Origin, P1 and P2 to be constrained within the volume bounds.
 * @param {*} planeToTransform
 * ...
 */
export function transformPlane(planeToTransform, targetPlane, viewType)
````